README.md

```diff
@@ -43,6 +43,9 @@ Or, import only a subset of the library:
 import isEmail from 'validator/lib/isEmail';
 ```
 
+#### Types 
+Starting the latest version of validator.js , the types comes out of box. 
```

Comment:
nit: perhaps can add a small sample typescript code block? this could be more beginner friendly.

Reply:
yeah will do that

Reply:
The wording should be corrected and also modified to make it "future safe".

e.g. what is "the latest version"? Currently, we know, but what if we read this README in e.g. 2 years, it will still say "the latest version" -> even though there may have been a lot of new versions released in between.

a bit of nitpicking from my side, sorry :-D

So my suggestion would be something more like:
```suggestion
TypeScript type definitions were previously not provided out-of-the-box, but got taken care of by third-parties like `DefinitelyTyped`. 
This is not required anymore, as type definitions are now part of validator.js
```